src/components/NcCounterBubble/NcCounterBubble.vue

```diff
@@ -243,7 +243,7 @@ export default {
 	methods: {
 		humanizeCount(count) {
 			if (this.raw) {
-				return count
+				return count.toString()
```

Comment:
I prefer this but this is also fine for me :)

```suggestion
				return String(count)
```